dev-util/nvidia-cuda-sdk/nvidia-cuda-sdk-8.0.44.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cuda eutils flag-o-matic portability toolchain-funcs unpacker versionator
+
+MYD=$(get_version_component_range 1-2)
+
+DESCRIPTION="NVIDIA CUDA Software Development Kit"
+HOMEPAGE="https://developer.nvidia.com/cuda-zone"
+SRC_URI="https://developer.nvidia.com/compute/cuda/${MYD}/prod/local_installers/cuda_${PV}_linux-run -> cuda_${PV}_linux.run"
+
+LICENSE="CUDPP"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux"
+IUSE="debug +doc +examples opencl +cuda"
+
+RDEPEND="
+	~dev-util/nvidia-cuda-toolkit-${PV}
+	media-libs/freeglut
+	examples? (
+		media-libs/freeimage
+		media-libs/glew:*
```

Comment:
I checked the compiled examples with ldd and greped for 'glew', which is not linked. Since glew headers are also shipped with GL, I assume that glew is linked statically. Thus a recompile when glew is updated shouldn't be needed. Please correct me if my assumptions are wrong.